config/extract-bootnodes

```diff
@@ -0,0 +1,71 @@
+#! /usr/bin/env python3
+
+"""This is a quick script that reads a nodes.json as generated by parity in
+order to create a list of possible bootstrap nodes.
+
+The nodes.json file can be found in
+~/.local/share/io.parity.ethereum/chains/Trustlines/network/nodes.json
+"""
+
+import sys
+import json
+import socket
+import re
+
+
+def try_socket_connect(address):
+    """try to connect to the given address
+
+    Some of the nodes listed in the nodes.json file are rather old. Try to
+    connect to them
+    """
+    sock = None
+    try:
+        sock = socket.create_connection(address, timeout=5.0)
+        return True
+    except (socket.error, socket.timeout):
+        return False
+    finally:
+        if sock is not None:
+            sock.close()
+
+
+def read_nodes_json(nodes_json_path):
+    with open(nodes_json_path, "rb") as f:
+        return json.load(f)["nodes"]
+
+
+def find_network_address(url):
+    match = re.match(".*@([.0-9]+):([0-9]+)", url)
+    return (match.group(1), int(match.group(2)))
+
+
+def main():
+    if len(sys.argv) != 2:
+        print("Usage: extract-bootnodes /path/to/nodes.json")
+        sys.exit(1)
+
+    nodes_json_path = sys.argv[1]
+    nodes = read_nodes_json(nodes_json_path)
+    nodes = [node for node in nodes if node["last_contact"].get("success", 0)]
```

Comment:
I tested it with all chains I have on my machine ever synced. Some map `"last_contact": null`. Therefore I get a `AttributeError` for the `NoneType`. In the end I don't get any nodes. I would say we should check this and ignore these nodes.